### PR TITLE
Created volunteers action test cases

### LINKED ISSUE #527: Volunteers Remove E2E Test
- When a user removes certain entries, then make sure the results are removed from the dataset

### LINKED ISSUE #533: Volunteers Actions E2E Test
- When applying different action buttons, the dataset should reflect accordingly

### PR DESCRIPTION
**Pull request by seanquiambao**
image

### CODE REVIEW

File: cypress/e2e/admin/volunteers/Actions.cy.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { volunteerList } from "../../../../src/data/mock/Volunteers";
+
+const firstFive = volunteerList.slice(0, 5);
```
Comment (shahdivyank) [inline cypress/e2e/admin/volunteers/Actions.cy.js]: call this just five

**Comment by seanquiambao** [comment]
wait i didn't realize that the delete test case was another separate issue, should i delete those

**Comment by shahdivyank** [comment]
Attach the issue for that as well

File: cypress/e2e/admin/volunteers/Actions.cy.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    five.forEach((volunteer) =>
+      cy
+        .get(`[data-cy="${volunteer.uid}"]`)
+        .find('[data-cy="confirm-tag"]')
+        .should("exist")
+    );
+  });
+
+  it("Not Attending First Five Entries", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/volunteers/Actions.cy.js]: Five -> 5

**Pull request merged**